src/drop-down-menu.jsx

```diff
@@ -65,6 +66,7 @@ let DropDownMenu = React.createClass({
 
   getStyles(){
     let zIndex = 5; // As AppBar
+    let disabled = this.props.disabled;
```

Comment:
Just to make sure we're on the same page...

The purpose of `disabled` is to make property value ternary evaluations of the object literal `styles` more terse. Because `this.props.disabled` is a `React.PropTypes.bool`, mutating `disabled` would not mutate `this.props.disabled`.

Therefore, are you saying that `disabled` should be declared with `const` instead of `let` to make it clear that `disabled` should not be mutated (even though doing so would not mutate `this.props`)?